.gitignore

```diff
@@ -9,7 +9,9 @@ oldeslintrc.js
 reactHooksSample.jsx
 coverage
 spec/__snapshot__
-Restaurants.csv
-Reviewers.csv
-Reviews.csv
+CSV/Restaurants.csv
+CSV/Reviewers.csv
+CSV/Reviews.csv
+CSV/ReviewRestaurantRelationships.csv
+CSV/ReviewReviewerRelationships.csv
```

Comment:
Good call on making a csv folder for all of these. Very clear what you're doing here!